mailpoet/assets/js/src/common/premium_key/key_messages/service_unavailable_messages.tsx

```diff
@@ -1,4 +1,4 @@
-import { MailPoet } from 'mailpoet';
+import { MailPoet } from '../../../mailpoet';
```

Comment:
I'm not sure this change was intentional. Normally we use `import { MailPoet } from 'mailpoet';`.

Reply:
I had this fixed already but forgot to push that commit with the fix before asking for the review. Apologies for the noise.